app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,47 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = current_user.foods
+  end
+
+  # GET /foods/new
+  def new
```

Comment:
- [ ] I think you forgot to fix n+1 problem as mentioned by the previous reviewer so I highly recommend making it. so, Let's optimize our database queries by introducing solutions to N + 1 problem in the code to improve the app's performance. For instance, instead of fetching all books here, the last code will fetch books and the associated authors in a single instead of making two different queries. This improves the performance.👏🏻💯🚀⭐

![image](https://github.com/MayPyone/Recipe-app/assets/58553711/78e1babc-cf9e-4687-8dcb-9e95f474e955)